getting-started-vue/05.md

```diff
@@ -2,9 +2,7 @@
 
 ### Step 12. Show completed tasks
 
-Out of the box, the `<ListView>` component detects a tap gesture for every item and emits an event for it. The event carries information about the index of the tapped array item and the array item itself. To let the user choose the outcome of a tap gesture and expand the functionality of your app, you can tie a dialog to the event.
-
-[`dialogs`](https://docs.nativescript.org/api-reference/modules/_ui_dialogs_) is a globally available module that provides several configurable dialog types for apps: alert, action, prompt, login, confirmation. This lesson explores the use of [`action()`](https://nativescript-vue.org/en/docs/elements/dialogs/action/) to let the user choose if they want to mark a task as completed or delete it from the list of active tasks.
+Create a similar list for the completed tasks
```

Comment:
Can you change this sentence up to add a bit more of a description of this step? Something like...

In this step you’ll create a similar list for your app’s Completed tab.

Reply:
Or we can just make that wording change when we land this @vtrifonov.